core/controllers/base.py

```diff
@@ -411,6 +411,16 @@ def vaidate_and_normalize_args(self):
             arg: normalized_value.get(arg) for arg in request_arg_keys
         }
 
+        default_value_keys = list(
+            set(normalized_value.keys()) -
```

Comment:
It took me a while to understand what normalized_value is. Should it be normalized_arg_values instead?

Reply:
Done